index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My_Microverse_Portfolio_Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <h1>Welcome</h1>
+      <button class="btn-trigger">
+        <img src="./assets/img/union.svg" alt="button"/>
+      </button>
+    </header>
+    <section class="headline">
+      <h2>
+        <span>Hey there. I’m Chimwemwe</span>
+        <span class="green-txt">I’m a software developer</span>
+      </h2>
+      <p class="supporting-txt">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <div class="social-media-icons">
+        <img src="./assets/img/twitter.svg" alt="">
+        <img src="./assets/img/Linkedin.svg" alt="">
+        <img src="./assets/img/medium.svg" alt="">
+        <img src="./assets/img/github.svg" alt="">
+        <img src="./assets/img/angellist.svg" alt="">
+      </div>
```

Comment:
- [ ] Kindly let's have these social media logos inside a list since it's a list of links, and remember they are supposed to be clickable, for this you cab use the unordered list `<ul>`, see the example below 👇 
```
<ul>
   <li>
        <a href="your social media link goes here"><img src="./assets/img/twitter.svg" alt=""></a>
   </li>
   <li>
        <a href="your social media link goes here"><img src="./assets/img/twitter.svg" alt=""></a>
   </li>
</ul>
```

This will keep our code more semantic 👍